src/__tests__/ParserComponent.test.tsx

```diff
@@ -36,51 +38,126 @@ describe('<ParserComponent /> renders correctly', () => {
   });
 });
 
-describe('Csv FileEvents processed correctly\n', () => {
+it('should trigger onChange event when file selected', async () => {
   const props = {
     prompt: 'test: ',
     fileType: FileType.csv,
   };
+  const onChangeMock = jest.fn();
+  const testFile: File = new File(
+      ['abcdef'],
+      'test.csv',
+      {type: '.csv,text/csv'},);
+
+  const event = {target: {files: [testFile]}};
+  const comp = mount(
+      <ParserComponent
+        {...props}
+        onChange={onChangeMock}
+      />);
+
+  comp.find('input').simulate('change', event);
+  const fileUsed: File = onChangeMock.mock.calls[0][0];
+  expect(fileUsed.name).toBe(testFile.name);
+  expect(onChangeMock).toHaveBeenCalledTimes(1);
+});
 
-  it('Onchange event triggered when file selected\n', async () => {
-    const onChangeMock = jest.fn();
-    const testFile: File = new File(
-        ['abcdef'],
-        'test.csv',
-        {type: '.csv,text/csv'},);
-
-    const event = {target: {files: [testFile]}};
-    const comp = mount(
-        <ParserComponent
-          {...props}
-          onChange={onChangeMock}
-        />);
+describe('R1 Tests\n', () => {
+  const csvProps = {
+    prompt: 'test: ',
+    fileType: FileType.csv,
+  };
 
-    comp.find('input').simulate('change', event);
-    const fileUsed: File = onChangeMock.mock.calls[0][0];
-    expect(fileUsed.name).toBe(testFile.name);
-    expect(onChangeMock).toHaveBeenCalledTimes(1);
-  });
+  // store original console.warn
+  const originalWarn = console.warn;
+
+  // array to store console output
+  const consoleOutput: any[] = [];
```

Comment:
WHy?

Reply:
forgot to remove this. it was just to make the console a little easier to read because momentjs kept printing a warning about a deprecated function (which should probably be addressed).
![image](https://user-images.githubusercontent.com/47048148/69763743-d3a48400-1133-11ea-9256-0c4938f5e7aa.png)

Reply:
left it in, but commented out.